docker-compose.yml

```diff
@@ -27,49 +27,35 @@ services:
       - ./api:/app
     ports:
       - 5000:3000
-    depends_on: 
+    depends_on:
       db:
         condition: service_healthy
-    environment: 
+    environment:
       REDIS_URL: redis://redis:6379/0
   web:
     build:
       context: ./web
       dockerfile: Dockerfile.dev
     restart: always
     command: yarn run dev
-    volumes: 
+    volumes:
       - ./web:/app
       - /app/node_modules
     ports:
       - 3000:3000
-    environment: 
+    environment:
       APP_ENV: development
       API_URL: http://localhost:5000
-      INFURA_PROJECT_ID: xxx
```

Comment:
don't put sensitive information in source code

Reply:
alright, I have deleted the sensitive information from docker-compose.yml